inst/private/python_ipc_pytave.m

```diff
@@ -59,17 +61,50 @@
 
   info.prelines = 0;
 
-  % TODO: bit wasteful to repeat this every call
-  headers = python_header();
-  pyexec(headers)
+  if isempty(have_headers)
+    pyexec(strjoin({'import sys',
+                    'import sympy',
+                    'import sympy as sp',
+                    'from sympy import __version__ as spver',
+                    'from sympy import *',
+                    'from sympy.logic.boolalg import Boolean, BooleanFunction',
+                    'from sympy.core.relational import Relational',
+                    '# temporary? for piecewise support',
+                    'from sympy.functions.elementary.piecewise import ExprCondPair',
+                    'from sympy.integrals.risch import NonElementaryIntegral',
+                    'from sympy.matrices.expressions.matexpr import MatrixElement',
+                    '# for hypergeometric',
+                    'from sympy.functions.special.hyper import TupleArg',
+                    'from sympy.utilities.iterables import uniq',
+                    'import copy',
+                    'import binascii',
+                    'import struct',
+                    'import codecs',
+                    'import xml.etree.ElementTree as ET',
+                    'from distutils.version import LooseVersion',
```

Comment:
are all these imports required? I merely copy pasted all of them because some tests showed error regarding a few of the imports..